### DISCUSSION TITLE
DS3 controller showing right stick and left trigger enganged when in SXS mode, controller behaves fine otherwise.

### DISCUSSION THREAD

**Discussion by dontblush** [nefarius/DsHidMini#368] [Q&A] [answered]
SXS mode is needed for pressure-sensitive games in PCSX2, but when the controller is put in the mode via DsHidMini, it shows the right stick being pushed down and it moves up when the left trigger is pressed. I've never had an issue with this program until literally today. When in any other mode the controller behaves just fine and still works on a PS3

**Comment by nefarius** [comment, marked as answer]
[All explained in the documentation](https://docs.nefarius.at/projects/DsHidMini/SCP-XInput-Bridge/#rpcs3-and-pcsx2-qt-edition).

**Reply by dontblush** [reply]
Double checked, thanks it worked.
